src/components/main/compose/messages/mod.rs

```diff
@@ -84,7 +85,7 @@ pub fn Messages(cx: Scope<Props>) -> Element {
     let chan = use_coroutine(&cx, |mut rx: UnboundedReceiver<ChanCmd>| async move {
         // used for timeouts
         let mut typing_times: HashMap<DID, Instant> = HashMap::new();
-        let mut prev_current_chat: Option<Uuid> = None;
+        let mut prev_current_chat: Option<ConversationInfo> = None;
```

Comment:
I believe that using ConversationInfo instead of Uuid will cause the TypingIndicator to trigger when we don't intend. If a conversation receives a new message, we may want to disable the typing indicator, but on line 100, the new and previous ConversationInfo structs will be different and this will trigger the typing indicator.